spec/factories/mentor_memberships.rb

```diff
@@ -0,0 +1,35 @@
+# == Schema Information
+#
+# Table name: mentor_memberships
+#
+#  id         :uuid             not null, primary key
+#  type       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  mentor_id  :uuid             not null
+#  school_id  :uuid             not null
+#
+# Indexes
+#
+#  index_mentor_memberships_on_mentor_id  (mentor_id)
+#  index_mentor_memberships_on_school_id  (school_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (mentor_id => mentors.id)
+#  fk_rails_...  (school_id => schools.id)
+#
+FactoryBot.define do
+  factory :mentor_membership do
+    association :school
+    association :mentor
+
+    trait :claims do
+      type { "Claims::MentorMembership" }
+    end
+
+    trait :placements do
+      type { "Placements::MentorMembership" }
+    end
```

Comment:
What about referencing the class itself? Rails accepts this for STI types – it doesn't have to be given a String.

```suggestion
    trait :claims do
      type { Claims::MentorMembership }
    end

    trait :placements do
      type { Placements::MentorMembership }
    end
```

Presumably this would give us some added intellisense in our IDEs. At the very least we'll be able to `Cmd/Ctrl + click` on the class name to jump to the definition.

No biggie, though. Feel free to skip this one if you don't think it's a valuable change.

Reply:
c06a141d8659b694ad202fd79569d29eaeceaf2c